lib/Runtime/Language/TaggedInt.inl

```diff
@@ -206,7 +206,23 @@ namespace Js
     }
 #endif
 
+    // Returns true when the provided args only contains TaggedInts. Note that
+    // iteration starts from 0 (to account for 'this' at index 0).
+    // ** Is this too large of a function to inline?
```

Comment:
You can use the REVIEW: prefix to comments in the future to indicate this (by convention) :)